src/MP4TagReader.js

```diff
@@ -208,6 +208,10 @@ class MP4TagReader extends MediaTagReader {
       var dataLength = atomSize - atomHeader;
       var atomData;
 
+      if (atomName === 'covr' && type === 'uint8') {
```

Comment:
Can you add a comment here explaining why we're doing this hack?

PS: thank you for the PR and sorry for taking so long to look into it.